<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alert</class>
 <widget class="QWidget" name="Alert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>This is Response!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/Icon.png</normaloff>../images/Icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Massage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Massage</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SubMassage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
